mlflow/server/js/.eslintrc.js

```diff
@@ -110,9 +113,7 @@ module.exports = {
     'import/prefer-default-export': 0,
     'import/unambiguous': 0,
     'indent': 0,
-    'indent-legacy': ['error', 2, {
-      'SwitchCase': 1,
-    }],
+    'indent-legacy': 0,
```

Comment:
Prettier indents the highlighted lines, which conflicts with [indent-legacy](https://eslint.org/docs/rules/indent-legacy)

<img width="416" alt="Screen Shot 2020-04-26 at 17 57 18" src="https://user-images.githubusercontent.com/17039389/80302824-65014a80-87e7-11ea-8cfc-86e3fb56b664.png">

Reply:
This should look like the following
```
    ? Promise.resolve()
    : wrapDeferred(Services.deleteModelVersion, {
      name: modelName,
      version: version,
    }),
```
Can we tune the prettier config to make indentation like this?

Reply:
I just checked the output. This is actually fine to be indented like:
```
export const deleteModelVersionApi = (modelName, version, id = getUUID(), localUpdateOnly) => ({
  type: DELETE_MODEL_VERSION,
  payload: localUpdateOnly
    ? Promise.resolve()
    : wrapDeferred(Services.deleteModelVersion, {
        name: modelName,
        version: version,
      }),
  meta: { id, modelName, version },
});
```